packages/sample/src/main.ts

```diff
@@ -6,6 +6,7 @@ const program = new Command();
 
 program.version('1.0.0')
     .option('-l --list', 'List connected devices')
+    .option('-a --analog', 'Run the analog demo')
```

Comment:
Should expose primitives, not a demo

Reply:
Converted it to take analog port as an input, along with flag to set continuous vs one-shot